contracts/polling/PollCreator.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.5.11;
+
+import "./Poll.sol";
+import "../token/ILivepeerToken.sol";
+
+
+contract PollCreator {
+    // TODO: Update these values
+    uint256 public constant QUORUM = 20;
+    uint256 public constant THRESHOLD = 50;
+    uint256 public constant POLL_PERIOD = 10 * 5760;
+
+    ILivepeerToken public token;
+    uint256 public pollCreationCost;
```

Comment:
Which one?
The bottom two (`token` and `pollCreationCost`) are being set in the constructor so they can not be evaluated at compile time.

Reply:
`pollCreationCost` could be a constant (I realize that my other comment in the review about adjusting the value in `migrations.config.js` actually wouldn't be needed then) since it'll be fixed upon deployment.

Reply:
fixed in bccafae